DittoDiskUsage/src/main/java/live/ditto/dittodiskusage/usecase/GetDiskUsageMetrics.kt

```diff
@@ -16,7 +16,7 @@ class GetDiskUsageMetrics {
     fun execute(currentState: DiskUsageState): HealthMetric {
 
         val dittoStoreSize: Int =
-            currentState.children.first { shortRelativePath(it.relativePath) == DITTO_STORE }.sizeInBytes
+            currentState.children.firstOrNull { shortRelativePath(it.relativePath) == DITTO_STORE }?.sizeInBytes ?: 0
```

Comment:
Huh, I thought this was fixed already, last time I was fighting with dependencies to review something

Reply:
This was basically saying it didn't find anything matching the predicate for `DITTO_STORE `, which means that the directory doesn't exist or maybe we're looking in the wrong place?

Doing a null check means that if it fails at least it won't crash.

Reply:
It might be worth revisiting this after SDK 4.9.0, there are some stability improvements around the disk usage tooling in the SDK.

This is a great answer for now then.